### PR TITLE
BusinessDashboard Theme | <SidePanel>  - Changed spacings and colors

### PR DESCRIPTION
**Pull request by shelly-goldblit**
### 🔦 Summary
SidePanel overrides + InfoIcon overrides (as it is an inherent part of side panel).
...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [x] 👨‍🎨 UX change is approved by the Design System UX Ayelet Livne 
- 📚 Change is documented
  - [x] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6971/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6971

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_6971/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_6971

File: packages/wix-style-react/src/Themes/businessDashboard/components/SidePanel/Content/Content.st.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+:import {
+  -st-from: '../../../foundations/colors.st.css';
+  -st-named: D40, D55;
+}
+
+.root {
+  -st-extends: SidePanelContent;
+  padding: 24px 20px 24px 20px;
```
Comment (adicicurel) [inline packages/wix-style-react/src/Themes/businessDashboard/components/SidePanel/Content/Content.st.css:13]: Use SP spacing

**Review by adicicurel — APPROVED** [review]

**Pull request merged**